packages/plugin-ext/src/plugin/command-registry.ts

```diff
@@ -86,3 +96,75 @@ export class CommandRegistryImpl implements CommandRegistryExt {
         }
     }
 }
+
+export class CommandsConverter {
```

Comment:
what is goal or logic of command converter ?

Reply:
added the comment